src/Microsoft.AspNet.Authentication.Cookies/CookieAuthenticationOptions.cs

```diff
@@ -146,7 +146,7 @@ public string CookieName
         /// An optional container in which to store the identity across requests. When used, only a session identifier is sent
         /// to the client. This can be used to mitigate potential problems with very large identities.
         /// </summary>
-        public IAuthenticationSessionStore SessionStore { get; set; }
+        public ITicketStore SessionStore { get; set; }
```

Comment:
Yeah we talked about that, but @Tratcher preferred the Session name to signal intent/usage for the property.